app/views/coronavirus_landing_page/components/landing_page/_live_stream_section.html.erb

```diff
@@ -1,3 +1,4 @@
+<% if live_stream["show_live_stream"] %>
```

Comment:
Would it be worth doing:
```ruby
<% if ActiveModel::Type::Boolean.new.cast(live_stream["show_live_stream"]) %>
```
Otherwise could setting `"show_live_stream": "false"` in the JSON have unexpected consequences?

Reply:
Nice catch, thanks

Reply:
I went with `live_stream["show_live_stream"] == true` to match the approach used elsewhere

Reply:
I think `"show_live_stream": false` will work as expected as `JSON.parse` will recognise it as the boolean. So the "false" option is a bit of an edge case. I just think it's an easy change to make, and makes the code a little more robust.

Reply:
OK, you've got me :-)